cmd/acr/login.go

```diff
@@ -62,23 +66,32 @@ func newLoginCmd(out io.Writer) *cobra.Command {
 	cmd.Flags().StringVarP(&opts.username, "username", "u", "", "the registry username")
 	cmd.Flags().StringVarP(&opts.password, "password", "p", "", "the registry password or identity token")
 	cmd.Flags().BoolVarP(&opts.fromStdin, "password-stdin", "", false, "read password or identity token from stdin")
+	cmd.Flags().BoolVarP(&opts.insecure, "insecure", "k", false, "allow connections to SSL registry without certs")
+	cmd.Flags().BoolVarP(&opts.plainHTTP, "plain-http", "", false, "allow insecure connections to registry without SSL")
+	cmd.Flags().BoolVarP(&opts.verbose, "verbose", "v", false, "verbose output")
```

Comment:
Adding those options are unplanned and are breaking changes. Let's remove the related code in this PR.

Reply:
Deleted these 3 lines and related code.